.travis.yml

```diff
@@ -1,9 +1,7 @@
 dist: xenial
-sudo: false
```

Comment:
I prefer to run with `sudo: false`. Is that the default now? If so, I'd like to make that change in jaraco/skeleton.

Reply:
The `sudo: false` env is no longer available and the `sudo` keyword is fully deprecated:

* https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Would you like me to remove this commit?

Reply:
Fine to leave it as submitted. Thanks for the reference.